tests/impl.cpp

```diff
@@ -2813,26 +2813,31 @@ result_t test_mm_sfence(const SSE2NEONTestImpl &impl, uint32_t iter)
 
 result_t test_mm_shuffle_pi16(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
+#if (__GNUC__ == 8 && __GNUC_MINOR__ == 3) || \
```

Comment:
I wonder if gcc versions other than 8.3 and 9.2 encounter the same problem or not.

Reply:
Is there any other version that suggested to check?

Reply:
> Is there any other version that suggested to check?

We have to clarify if the known issue appears in gcc version prior to 8.3.

Reply:
I have added the supported versions in README. And I found even g++-8.4 would meet the same compiling error, therefore I have made the corresponding change in `impl.cpp`